src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -622,4 +642,68 @@ public static void updateConfigErrorMessage(Config result, String key, String ms
       }
     }
   }
+
+  public static Map<String, String> getSchemaFromSchemaRegistry(
+      final String topicName, final String schemaRegistryURL) {
+    return getSchemaFromSchemaRegistry(topicName, schemaRegistryURL, "value");
+  }
+
+  public static Map<String, String> getSchemaFromSchemaRegistry(
+      final String topicName, final String schemaRegistryURL, final String type) {
+    Map<String, String> srConfig = new HashMap<>();
+    srConfig.put("schema.registry.url", schemaRegistryURL);
+    AvroConverterConfig avroConverterConfig = new AvroConverterConfig(srConfig);
```

Comment:
for my understanding, does providing schema registry mean the converter will always be avro?

Reply:
Not really. It could also be in JsonSchema or in Protobuf. I didn't find any usage case in our code though. Maybe it isn't a safe assumption.

Reply:
I am okay with the limitation that schematization only works with avro right now, but can we mention it on top of the method?
There are two ways to tackle this. 
1. The config which customer passes has the value.converter 
   a. You can check before fetching schema from SR. 
2. If they dont have value.converter, it is likely they are fetching it from connect distributed properties value and you will get null from value.converter. 
   a. In this case, you will have to rely on calling the SR from the URL and hope it returns else you can say Schematization is not supported for non avro converters.

Reply:
bump..